package.json

```diff
@@ -15,7 +15,10 @@
     "test": "npm run eslint && npm run prettier"
   },
   "dependencies": {
+    "@emotion/react": "^11.10.6",
+    "@emotion/styled": "^11.10.6",
     "@headlessui/react": "^1.7.7",
+    "@mui/material": "^5.11.10",
```

Comment:
Another option, if you feel like trying it, would be to replace `@headlessui/react` with `radix-ui`: https://www.radix-ui.com/
It's a library that we would like to try and it includes a tooltip component. Your feedback about it would be welcome!